src/mote/modules/late.py

```diff
@@ -25,12 +25,14 @@
 from datetime import datetime
 from time import ctime
 
+from flask import current_app as app
+
 seconds_delta = 86400
 
 
 def fetch_recent_meetings(days):
     try:
-        datagrepper_base_url = "https://apps.fedoraproject.org"
+        datagrepper_base_url = app.config["DATAGREPPER_BASE_URL"]
```

Comment:
same remark as `recognition_pattern`: you can remove this line and use `app.config["DATAGREPPER_BASE_URL"]` directly